### PR TITLE
llvmPackages_11: 11.0.1 -> 11.1.0

### PR DESCRIPTION
**Pull request by ggreif**
###### Motivation for this change

https://github.com/llvm/llvm-project/releases/tag/llvmorg-11.1.0
https://lists.llvm.org/pipermail/release-testers/2021-February/001472.html
https://lists.llvm.org/pipermail/release-testers/2021-January/001466.html
https://lists.llvm.org/pipermail/release-testers/2021-January/001457.html

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ggreif** [comment]
@GrahamcOfBorg build lldb_11

File: pkgs/development/compilers/llvm/10/openmp.nix [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{ stdenv
+{ lib, stdenv
```
Comment (SuperSandro2000) [inline pkgs/development/compilers/llvm/10/openmp.nix:1]:
```suggestion
{ lib
, stdenv
```

File: pkgs/development/compilers/llvm/5/openmp.nix [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{ stdenv
+{ lib, stdenv
```
Comment (SuperSandro2000) [inline pkgs/development/compilers/llvm/5/openmp.nix:1]:
```suggestion
{ lib
, stdenv
```

File: pkgs/development/compilers/llvm/6/openmp.nix [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{ stdenv
+{ lib, stdenv
```
Comment (SuperSandro2000) [inline pkgs/development/compilers/llvm/6/openmp.nix:1]:
```suggestion
{ lib
, stdenv
```

File: pkgs/development/compilers/llvm/8/openmp.nix [hunk 4]
```diff
@@ -1,4 +1,4 @@
-{ stdenv
+{ lib, stdenv
```
Comment (SuperSandro2000) [inline pkgs/development/compilers/llvm/8/openmp.nix:1]:
```suggestion
{ lib
, stdenv
```

File: pkgs/development/compilers/llvm/9/openmp.nix [hunk 5]
```diff
@@ -1,4 +1,4 @@
-{ stdenv
+{ lib, stdenv
```
Comment (SuperSandro2000) [inline pkgs/development/compilers/llvm/9/openmp.nix:1]:
```suggestion
{ lib
, stdenv
```

**Comment by bobrik** [comment]
11.1.0 is out now

**Comment by bobrik** [comment]
Don't you need to update the checksums as well?

**Comment by ggreif** [comment]
@bobrik Sure, my Mac needs to  build a lot. Patience!

**Comment by SuperSandro2000** [comment]
@ofborg eval

🤔 
```
Automatic merge went well; stopped before committing as requested
error: --- TypeError ------------------------------------------------------------------------------------------------------------------------------------------------------------------- nixat: (12:35) in file: /nix/store/lhz31hg5xsz65acfi2a47kv05gyd36dx-nixpkgs-review/lib/python3.8/site-packages/nixpkgs_review/nix/evalAttrs.nix

    11|     pkg = lib.attrByPath attrPath null pkgs;
    12|     maybePath = builtins.tryEval "${pkg}";
      |                                   ^
    13|   in rec {

cannot coerce a set to a string
```

**Comment by ggreif** [comment]
@SuperSandro2000 Sorry, that doesn't ring a bell...
@bobrik Unfortunately this laptop doesn't quite build `staging`, so I am a bit lost with the rest  of the hashes. Feel free to build on this. Also why do the aarch64 builders present hash mismatches in a base64-like way and intel ones in a base32 style?

**Comment by SuperSandro2000** [comment]
> Unfortunately this laptop doesn't quite build `staging`, so I am a bit lost with the rest of the hashes. Feel free to build on this. Also why do the aarch64 builders present hash mismatches in a base64-like way and intel ones in a base32 style?

If you can list the packages I should build I can check if there are more has mismatches.

**Comment by primeos** [comment]
> @bobrik Unfortunately this laptop doesn't quite build staging, so I am a bit lost with the rest of the hashes.

It should usually be fine if you test this on e.g. the current `nixos-unstable` channel revision ;)

> I should have them all now.

LGTM as well :) Thanks!

> If you can list the packages I should build I can check if there are more has mismatches.

Good idea. I've just posted my silly script here in case anyone ever needs to verify the hashes again: https://github.com/NixOS/nixpkgs/issues/100725#issuecomment-781323979

**Review by primeos — APPROVED** [review]
The diff LGTM and `llvmPackages_11 on x86_64-linux — Success` (CI) so this should be good to go :)

**Review by bobrik — APPROVED** [review]

**Pull request merged**